frontend/app/src/App.vue

```diff
@@ -1,41 +1,82 @@
 <template>
   <v-app id="rotki">
-    <div v-show="logged">
+    <div show="logged" class="app__content rotkiLightGrey">
```

Comment:
Would it be easy to change the css class (+ color in the vuetify.ts I am guessing) to kebab-case instead of camelCase?

Reply:
I have to check if this will work everywhere... Technically it's not just a class (e.g. it's also used in cases like this `(var(--v-primary-base)`

Reply:
Fixed